File: Selenium/Project/Python_Project/Activity8.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC

# Initialize the WebDriver
driver = webdriver.Chrome()

try:
    # Navigate to the website's backend login page
    driver.get('https://alchemy.hguy.co/jobs/wp-admin/')
    
    # Find the username field of the login form and enter the username
    username_field = WebDriverWait(driver, 10).until(
        EC.presence_of_element_located((By.ID, "user_login"))
    )
    username_field.send_keys("root")  # Enter the username
    
    # Find the password field of the login form and enter the password
    password_field = driver.find_element(By.ID,"user_pass")
    password_field.send_keys("pa$$w0rd")  # Enter the password
    
    # Find the login button and click it
    login_button = driver.find_element(By.ID,"wp-submit")
    login_button.click()
    
    # Verify that you have logged in
    WebDriverWait(driver, 10).until(EC.title_contains("Dashboard"))
    assert "Dashboard" in driver.title, "Login failed"
    print("Logged in successfully.")

finally:
    # Close the browser
    driver.quit()

File: Selenium/Project/Project_Java/Activity8.java
package Project;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

import java.time.Duration;

public class Activity8 {
    public static void main(String[] args) {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
// Navigate to the site’s backend
        driver.get("https://alchemy.hguy.co/jobs/wp-admin");

        // Find the username field of the login form and enter the username
        WebElement usernameField = driver.findElement(By.id("user_login"));
        usernameField.sendKeys("root");

        // Find the password field of the login form and enter the password
        WebElement passwordField = driver.findElement(By.id("user_pass"));
        passwordField.sendKeys("pa$$w0rd");

        // Find the login button and click it
        WebElement loginButton = driver.findElement(By.id("wp-submit"));
        loginButton.click();

        // Verify that you have logged in
        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
        wait.until(ExpectedConditions.titleContains("Dashboard"));

        if (driver.getTitle().contains("Dashboard")) {
            System.out.println("Logged in successfully.");
        } else {
            System.out.println("Login failed.");
        }

        // Close the browser
        driver.quit();
    }
}